### DISCUSSION TITLE
eventResize

### DISCUSSION THREAD

**Discussion by invaders-xx** [saade/filament-fullcalendar#39] [Ideas]
The plugin does not handle eventResize. Right ? it possible to add it ? I can make a PR if you want ? and also change record and record_id to event and event_id ?

**Comment by saade** [comment]
PR are always welcome! 

As i said, i only use the calendar for displaying static data, so people with more use-cases can help build a better pluggin.



> and also change record and record_id to event and event_id ?
- yes but do it in a sepparated PRs

**Reply by invaders-xx** [reply]
2 PR's created ;-)
